midp/background.js

```diff
@@ -66,7 +66,7 @@ Native["com/nokia/mid/s40/bg/BGUtils.waitUserInteraction.()V"] = function() {
       localmsgServerWait = resolve;
     });
   }).then(function() {
-    document.getElementById("splash-screen").style.display = "block";
-    document.getElementById("background-screen").style.display = "none";
+    showSplashScreen();
```

Comment:
*appendChild* should remove the child from the parent and then append it again, but I guess that's not going to be noticeable.